ere-guests/stateless-validator/reth/guest/Cargo.toml

```diff
@@ -26,6 +26,11 @@ alloy-primitives.workspace = true
 
 # external
 k256.workspace = true
-bincode.workspace = true
+bincode_v2.workspace = true
 serde.workspace = true
 serde_with.workspace = true
+once_cell.workspace = true
+
+[features]
+default = ["std", "reth-stateless/default"]
```

Comment:
Need `reth-stateless/default` enables `k256`.

Also thought about requiring the feature `k256` in each guest crate (which is more straightforward), but that seems easy to forget to add.